### PR TITLE
[Nova] Fix Linux Wheels CUDA/ROCm Versioning Issue

### PR DESCRIPTION
**Pull request by osalpekar**
Pass CU_VERSION correctly in Linux Wheels builds. We are already doing this in conda builds, and this doesn't apply to Mac builds, so Linux Wheels was the only affected workflow.

### CODE REVIEW

File: .github/workflows/build_wheels_linux.yml [hunk 1]
```diff
@@ -84,6 +85,7 @@ jobs:
       - name: Install torch dependency
         run: |
           source "${BUILD_ENV_FILE}"
+          set -ex
```
Comment (weiwangmeta) [inline .github/workflows/build_wheels_linux.yml:88]: Curious to know what difference does this make? thx!

**Review by weiwangmeta — APPROVED** [review]
The fix looks good!

**Review by atalman — APPROVED** [review]
LGTM

File: .github/workflows/build_wheels_linux.yml (hunk 1 above)
Comment (osalpekar) [reply .github/workflows/build_wheels_linux.yml:88]: -x will print the full bash command that is run (good for debuggability, and ideally we should do so for every step in each workflow)
-e will make the script exit upon first failure.

Comment (weiwangmeta) [reply .github/workflows/build_wheels_linux.yml:88]: thanks for the info.

**Pull request merged**